dbms/src/Common/OptimizedRegularExpression.inl.h

```diff
@@ -26,8 +25,15 @@
 #include <optional>
 
 #define MIN_LENGTH_FOR_STRSTR 3
-#define MAX_SUBPATTERNS 5
+#define MAX_SUBPATTERNS 10
```

Comment:
Is this value must be the same as `max_captures`?

Reply:
> Is this value must be the same as `max_captures`?

Yes, and I will combine them together.